generators/client/templates/react/_package.json

```diff
@@ -43,6 +43,7 @@ limitations under the License.
     "react-redux-loading-bar": "2.9.3",
     "react-router-dom": "4.2.2",
     "react-router-modal": "1.2.0",
+    "react-time-format": "0.0.5",
```

Comment:
can we use https://github.com/andreypopp/react-time instead as it has more stars and hence more users. Another option is to use moment directly as we already have it on the path. But lest go with the lib as its more elegant

Reply:
I tried https://github.com/andreypopp/react-time but when the date is not good it writes "Invalid dates" (https://github.com/andreypopp/react-time/blob/master/src/__tests__/Time-test.js) so we should test the return before to display, so I think react-time-format is better

Reply:
what do you mean by date is not good? do you mean invalid date or invalid format? in either case if the date is invalid, I think its ok to show that, my main issue with react-time-format has only 4 stars and we dont want to depend on a lib that has too little user base. I would rather just use moment directly to format the date  something like 
`<dd>{moment(user.createdDate).format(DD/MM/YY hh:mm)}</dd>`

Reply:
The users created by default like 'admin' haven't the property 'lastModifiedDate' so it displays 'Invalid date' in all languages ...

Reply:
Oh that case, its a valid case and you should just do a truthy check
`<dd>{user.createdDate ? <Time value={user.createdDate} format={APP_DATE_FORMAT} /> :
 null}</dd>`

This is quite common in React, IMO printing empty for invalid date by a lib is not correct, coz this can lead to unintentional bugs which will go unnoticed when some data is missing and by doing the truthy check developer explicitly has control

Reply:
Aother option is to write our own wrapper which does the format using momnet and use that component. I guess it will be a very simple componenet. We can have it in the react-jhipster lib https://github.com/jhipster/react-jhipster/